### PR TITLE
Fix release documentation and fix generation of release notes

### PR DESCRIPTION
**Pull request by vojtapolasek**
#### Description:

Fix wrong or not exact parts of release documentation.
During generation of release notes, files ending in ..rofile within profile_stability folder are skipped.

#### Rationale:

The  release documentation was not exact.
If there were any changes done to profile files within profile_stability folder, they caused an exception during generation of release notes.

### CODE REVIEW

**Comment by mildas** [comment]
**Changes identified:**
Others:
 Python abstract syntax tree change found in release_tools/content_gh.py. - ctest will be selected.

**Recommended tests to execute:**
 (cd build && ctest -j4)

**Comment by mildas** [comment]
**Changes identified:**
Others:
 Python abstract syntax tree change found in release_tools/content_gh.py. - ctest will be selected.

**Recommended tests to execute:**
 (cd build && ctest -j4)

File: release_tools/README.adoc [hunk 1]
```diff
@@ -21,24 +21,26 @@ authentication tokens for these services.
...
 
-* Create `.env` file in `release_tools` directory, to hold your tokens:
+* Create `.env.yml` file in `release_tools` directory, to hold your tokens:
 +
-   GITHUB_TOKEN='<your github token here>'
-   JENKINS_USER='<you jenkins user ID>'
-   JENKINS_TOKEN='<you jenkins token here>'
+   github_token: '<your github token here>'
+   jenkins_user: '<you jenkins user ID>'
+   jenkins_token: '<you jenkins token here>'
```
Comment (yuumasato) [inline release_tools/README.adoc:30]: Just to be sure, the keys here are not case sensitive?

File: release_tools/README.adoc [hunk 2]
```diff
@@ -21,24 +21,26 @@ authentication tokens for these services.
 * Your Jenkins user - your User ID is probably the same as your GitHub handle, to be sure,
   check your *profile page* in Jenkins.
 * A Jenkins token - generate a token on *Configure page* in Jenkins
+  While generating the token, make sure that your Jenkins user has build permissions in Jenkins.
```
Comment (yuumasato) [inline release_tools/README.adoc:24]: The users don't have power to grant permissions to themselves.
I guess the message here is that not everyone can start builds from the job page, and they need permission to do so in order to use the automation scripts.

File: release_tools/README.adoc (hunk 1 above)
Comment (vojtapolasek) [reply release_tools/README.adoc:30]: I think they are not, this is derived from my current setup which I used during last release.

**Comment by mildas** [comment]
**Changes identified:**
Others:
 Python abstract syntax tree change found in release_tools/content_gh.py. - ctest will be selected.

**Recommended tests to execute:**
 (cd build && ctest -j4)

File: release_tools/README.adoc (hunk 1 above)
Comment (yuumasato) [reply release_tools/README.adoc:29]: Okay, thanks

**Comment by yuumasato** [comment]
Thank you for updating the docs, :)

**Pull request merged**